vuepress/docs/docs/client-codegen.md

```diff
@@ -83,13 +83,14 @@ The settings available on the `cs` (`CalibanSettings`) builder are:
 - `enableFmt`: Enable code formatting with scalafmt (default: `true`).
 - `extensibleEnums`: Generate a fallback case class for unknown enum values (default: `false`).
 - `headers` (only defined for `url` settings): Supply extra headers when fetching the schema from a URL.
+- `excludeDeprecated`: Exclude fields and enum values with deprecated directive (default: `false`).
```

Comment:
Thanks for updating the docs! Do you mind doing it in a separate PR, because docs are published right away and it would be better to update them after this is released 🙏

Reply:
No problem 👍 Probably on thu-sunday this week.

Reply:
PR for docs https://github.com/ghostdogpr/caliban/pull/2171